src/test/scala/firrtlTests/DriverSpec.scala

```diff
@@ -76,21 +76,18 @@ class DriverSpec extends FreeSpec with Matchers with BackendCompilationUtilities
       val optionsManager = new ExecutionOptionsManager("test")
 
       optionsManager.parse(Array("--top-name", "dog", "fox", "tardigrade", "stomatopod")) should be(true)
-      println(s"programArgs ${optionsManager.commonOptions.programArgs}")
```

Comment:
These three `println`s are perhaps useful. Consider using:

```scala
info(s"""programArgs: ${optionsManager.commonOptions.programArgs.mkString(", ")}""")
```

To produce the following output:

```
[info] - options include by default a list of strings that are returned in commonOptions.programArgs
[info]   + programArgs: fox, tardigrade, stomatopod 
```